content/css/concepts/animations/animations.md

```diff
@@ -20,22 +20,31 @@ To use CSS animation, some keyframes for the animation must be specified.
 
 Keyframes hold what CSS styles the element will have at certain times. When specifying styles inside the `@keyframes` rule, the animation will gradually change from the current styles to the new styles.
 
-To get an animation to work, the animation must be bound to an element and the `animation-duration` should be defined. In this example code, the `<h1>` element will load as yellow and gradually change to orange over the course of 5 seconds.
+To get an animation to work, the animation must be bound to an element and the `animation-duration` should be defined. In this example code, the `<h1>` element will load as purple and gradually change to yellow over the course of 5 seconds.
 
 ```css
 /* The animation */
 @keyframes color-change {
   from {
+    color: purple;
+  }
+  25% {
     color: yellow;
   }
+  50% {
```

Comment:
With this edit, "yellow" appears twice in the animation. Is that intentional?

Reply:
@SSwiniarski I think that's the edit that was requested in the [linked issue](https://github.com/Codecademy/docs/issues/438).

Reply:
@KTom101 That's how the suggested edit was written, but I think the actual issue was just having an animation with multiple steps. I just think having yellow twice is odd.

Reply:
@SSwiniarski I see what you mean.

Reply:
@SSwiniarski @KTom101 How about changing yellow -> orange on line 32?

Reply:
@yangc95, That would be good.